### PR TITLE
Updates plugin generator and adds integration test

### PR DESCRIPTION
**Pull request by jamonholmgren**
Fixes issue #1418, updates the generated Ignite plugin to latest specifications, adds some integration tests to ensure we don't break this in the future.

image

### CODE REVIEW

File: src/templates/plugin/package.json.ejs [hunk 1]
```diff
@@ -10,23 +10,24 @@
...
-    "babel-eslint": "^7.1.1",
-    "np": "^2.12.0",
-    "nyc": "^10.1.2",
-    "sinon": "^1.17.7",
-    "standard": "^8.6.0"
+    "ava": "^1.4.1",
+    "babel-eslint": "^10.0.1",
+    "np": "^5.0.1",
+    "sinon": "^7.3.2",
+    "standard": "^12.0.1",
```
Comment (robinheinze) [inline src/templates/plugin/package.json.ejs]: Do we even still use standard? I thought we had taken it out everywhere.

**Comment by robinheinze** [comment]
Nice work! 💯

File: src/templates/plugin/package.json.ejs (hunk 1 above)
Comment (jamonholmgren) [reply src/templates/plugin/package.json.ejs]: We use tslint-config-standard on our other projects. This isn't TypeScript though.

**Pull request merged**